test/test_linalg.py

```diff
@@ -7453,6 +7453,11 @@ def test_preferred_linalg_library(self):
         self.assertEqual(out_ref, out1.cpu())
         self.assertEqual(out1, out2)
 
+    def test_permute_matmual(self):
```

Comment:
```suggestion
    def test_permute_matmul(self):
```

Reply:
Good catch, thank you!

Reply:
Yeah this happened to me on mualtiple occasions

Reply:
haha, just got your huamor :)